requirements.txt

```
@@ -4,9 +4,9 @@
 #
 #    pip-compile --generate-hashes --output-file=requirements.txt requirements.in
 #
---only-binary :all:
 --no-binary grapheme
 --no-binary grpclib
+--only-binary :all:
```

```suggestion
--only-binary :all:
--no-binary grapheme
--no-binary grpclib
```
See:
* https://github.com/python/release-tools/pull/130/files#r1627044096
* #133